pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -305,6 +312,12 @@
       public static final String CONFIG_OF_CONTROLLER_HTTPS_PORT = "controller.port";
       public static final String CONFIG_OF_SEGMENT_UPLOAD_REQUEST_TIMEOUT_MS = "upload.request.timeout.ms";
 
+      /**
+       * Service token for accessing protected controller APIs.
```

Comment:
Hmmm what is the difference between this config and the config in line 243 (and then in 371)? The comments all say the same thing :)

Reply:
all these do the same thing: authenticate with controller APIs. The only need for their replication is due to pinot's handling of configuration subsets. segment fetcher, uploader, etc. all manage their token separately.